docs/audit_report/src/02_security_issues.rst

```diff
@@ -1,7 +1,5 @@
 Security and Vulnerabilities
 ============================
 
```

Comment:
Not here, but in the rendered audit_report page 13: 4.2.3 Relevant Patches. The table seem to overflow the footer:
![image](https://github.com/sehlen-bsi/botan-docs/assets/126883116/edba0630-03a9-403f-bb88-95c37f03938e)

Also, there is an unnecessary blank page before this table.

Reply:
Oh crap.

Reply:
[Got it](https://github.com/sehlen-bsi/botan-docs/pull/143/commits/2941b1a021e4f06e37fc1cf113e250fd24e04298)! The Sphinx documentation states that it automatically renders tables with more than 30 rows as "longtable". Longtables are allowed to break across pages (and we've seen that before in the 3.0.0 document). Unfortunately, this particular table only had 27 rows.

Now I forced all of those patch tables to be longtables.